strings.json

```diff
@@ -12,10 +12,9 @@
     "language/comments": "Comments",
     "language/conditionals": "Conditionals",
     "language/doctype": "Doctype",
-    "language/extends": "Extends",
     "language/filters": "Filters",
     "language/includes": "Includes",
-    "language/inheritance": "Inheritance",
+    "language/inheritance": "Inheritance: Extends and Block",
```

Comment:
I renamed this so you can easily do Cmd+F on the page.

I verified in the browser that the sidebar handles over-long entries by wrapping them, so this doesn't break the layout.